users/zeineldeen/models/lm/external_lm_decoder.py

```diff
@@ -365,6 +368,9 @@ def _create_external_lm_net(self) -> dict:
       fusion_str += ' - {} * safe_log(source(2))'.format(self.prior_lm_opts['scale'])
       fusion_source += [ilm_decoder.output_prob_name]
 
+    if self.ext_lm_opts.get("local_norm", False):
+      fusion_str = fusion_str + '- tf.math.reduce_logsumexp(' + fusion_str + ", axis=-1, keepdims=True)"
```

Comment:
maybe this can be simpler like this:
```
fusion_str = fusion_str + f" - tf.math.reduce_logsumexp({fusion_str}, axis=-1, keepdims=True)"
```
(there was also no space after `-`)

Reply:
sure. or even
```suggestion
      fusion_str = f"{fusion_str} - tf.math.reduce_logsumexp({fusion_str}, axis=-1, keepdims=True)"
```